packages/abi-coder/src/utils/constants.ts

```diff
@@ -25,9 +25,11 @@ export const genericRegEx = /^generic (?<name>\w+)$/;
 /**
  * Encoding versions
  */
-export type EncodingVersion = '0' | '1';
 export const ENCODING_V0 = '0';
 export const ENCODING_V1 = '1';
+export const DEFAULT_ENCODING_VERSION = ENCODING_V0;
```

Comment:
Is the default always going to be `v0` ? If so then I don't think we need this constant and then @nedsalk 's [comment applies](https://github.com/FuelLabs/fuels-ts/pull/2126/files#r1574608969)